src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/_WelcomeWizard.cshtml

```diff
@@ -112,16 +112,18 @@
                         <legend class="heading-small">Are you going to sign the agreement?</legend>
                         <p>This allows you to spend funds.</p>
                         <label class="block-label selection-button-radio" for="add-agreements-1" data-target="guide-wizard-step-agreements">
-                            @{int value = 1;string check = ""; if (Model.Data.RequiresAgreementSigning > 0) { value = 1; } else { value = 2; check = "checked"; } }
+                            @{  bool agreementToSign = Model.Data.RequiresAgreementSigning > 0;
+                                int value = agreementToSign ? 1 : 2;
+                                string check = agreementToSign ? "" : "checked";}
                             <input id="add-agreements-1" type="radio" name="agreements-group" value="@value" checked=@check>
                             Yes
                         </label>
-                        <label class="block-label selection-button-radio" for="add-agreements-2" data-target="value-setup">
+                        <label class="block-label selection-button-radio" for="add-agreements-2">
                             <input id="add-agreements-2" type="radio" name="agreements-group" value="2">
 
                             No, someone else will sign the agreement
                         </label>
-                        @if (Model.Data.RequiresAgreementSigning > 0)
+                        @if (agreementToSign)
```

Comment:
Can we make "agreementToSign" a property on the model and just set the other values based on that